sql/schema_changer_test.go

```diff
@@ -510,6 +510,7 @@ func runSchemaChangeWithOperations(
 // that run simultaneously.
 func TestRaceWithBackfill(t *testing.T) {
 	defer leaktest.AfterTest(t)()
+	t.Skip("#7628")
```

Comment:
Will do if the CI eats it. Otherwise I'm not convinced that the assignee needs to be here (as the assignee of an issue can freely change and in fact should for test failures based on who's available).